parsers/manifest_parser.go

```diff
@@ -262,6 +257,15 @@ func (dm *YAMLParser) ComposeActions(mani *ManifestYAML, manipath string) (ar []
 				}
 
 				wskaction.Exec.Kind = kind
+
+				action.Location = filePath
+				dat, err := utils.Read(filePath)
+				utils.Check(err)
+				code := string(dat)
+                		if ext == ".zip" || ext == ".jar" {
```

Comment:
@pritidesai Did we want to include a JAR test case?  It would be a good idea for coverage; please open another issue if you do not want to fix here.